README.md

```diff
@@ -210,7 +210,29 @@ _main_ and merged to this latter once the feature is ready.
 Every time a new version is about to be released, a merge from your new branch to _main_
 will create a new tag and a new package in the package registry.
 
```

Comment:
Can you add a quick explanation of what your diagrams show us please ?

Reply:
Done